docs/topics/api/signing.rst

```diff
@@ -19,6 +19,10 @@ you may wish to use the
 `jpm sign <https://developer.mozilla.org/en-US/Add-ons/SDK/Tools/jpm#jpm_sign>`_
 command to interact with the signing API.
 
+If you are using ``curl`` to interact with the API you should be sure to pass
+the ``-g`` flag to skip "URL globbing" which won't interact well with GUIDs
```

Comment:
I think we should avoid calling them GUIDs to be consistent with #968, I think throughout the validator and docs they are called `add-on id`.